src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -72,11 +73,24 @@
            [quo/text (subs literal 0 (dec (count literal)))]])
     blocks))
 
+(defn add-edited-tag [content]
+  (update (:parsed-text content)
+          (dec (count (:parsed-text content)))
+          (fn [last-literal]
+            (update last-literal
+                    :children
+                    conj
+                    {:literal [quo/text (style/edited-style)
+                               (str " (" (i18n/label :t/edited) ")")]
+                     :type    :edited}))))
+
 (defn render-parsed-text
-  [{:keys [content]}]
+  [{:keys [content edited-at]}]
   (reduce render-block
           [:<>]
-          (:parsed-text content)))
+          (if edited-at
+            (add-edited-tag content)
```

Comment:
```clojure
(cond-> (:parsed-text content)
                 edited-at
                  add-edited-tag)
```

Reply:
This is literally what `cond->` was created for :)